Content.Server/Database/ServerDbManager.cs

```diff
@@ -86,131 +48,360 @@ public void Init()
                 builder.AddProvider(_msLogProvider);
             });
 
+            ServerDbContext = CreateDbContext();
+
+            ServerDbContext.Database.Migrate();
```

Comment:
Do not do DB migration synchronous in the `Init()` method. It causes multiple extra seconds of startup time because EFCore has high startup overhead.